notebooks/text_classification/solutions/text_generation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>This is very clear! Nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>